### PR TITLE
Lego 802

### PR DESCRIPTION
**Pull request by Shweta-Deshpande**

### CODE REVIEW

File: library/pom.xml [hunk 1]
```diff
@@ -1803,12 +1812,12 @@
                 </property>
             </activation>
             <properties>
-                <sonar.branch.name>${env.BRANCH_NAME}</sonar.branch.name>
+                <sonar.branch.name>${GIT_BRANCH}</sonar.branch.name>
```
Comment (johngmyers) [inline library/pom.xml:1815]: Shouldn't this be `env.GIT_BRANCH`?

**Comment by johngmyers** [comment]
Could you rebase and squash?

File: library/pom.xml (hunk 1 above)
Comment (Shweta-Deshpande) [reply library/pom.xml:1815]: It does not work with the "env." prefix.

**Comment by Shweta-Deshpande** [comment]
Creating a new PR.

**Pull request closed without merging**